exercicios/para-casa/entregas/maria-aparecida/bankAccount.js

```diff
@@ -0,0 +1,85 @@
+class BankAccount {
+    client;
+    bank;
+    accountNumber;
+    agencyNumber;
+    #disable = false;
+    #amount = 0;
+    #qtyDrawMoney = 0;
+    rateForDraw;
+  
+    constructor(client, bank, accountNumber, agencyNumber, rateForDraw) {
+      this.client = client;
+      this.bank = bank;
+      this.accountNumber = accountNumber;
+      this.agencyNumber = agencyNumber;
+      this.rateForDraw = rateForDraw;
+    }
+  
+    credit(amount) {
+      this.#amount += amount;
+      console.log(`O saldo atual é de R$ ${this.#amount}`);
+    }
+  
+    debit(amount) {
+      this.#amount -= amount;
+      console.log(`O saldo atual é de R$ ${this.#amount}`);
+    }
+  
+    transferTo(anotherAccount, amount) {
+      if (this.#amount < amount) {
+        throw "ERRO!!! Você não possui saldo suficiente para realizar essa operação";
```

Comment:
Ao rodar na minha máquina , não aparece erro o programa passa tranquilo, porém aparece uma sugestão de Expected an error object to be thrown - Esperava que um objeto de erro fosse lançado.

Para tirar essa observação de boa pratica sugiro que seja feito da seguinte forma: 

throw new Error("ERRO!!! Você não possui saldo suficiente para realizar essa operação")